src/client/todos/todo.styl

```diff
@@ -0,0 +1,26 @@
+.todo-item
+  font-size .875em
+  margin-bottom .25em
+  max-width 42em
+
+  &__label, &__button
```

Comment:
Debemify it to what...? .todo-item label, .todo-item button? Or .todo-item-label and .todo-item-button?